src/adapters/bidderFactory.js

```diff
@@ -323,15 +344,22 @@ export const processBidderRequests = hook('sync', function (spec, bids, bidderRe
       };
       onResponse(response);
 
-      let bids;
       try {
-        bids = spec.interpretResponse(response, request);
+        response = spec.interpretResponse(response, request);
       } catch (err) {
         logError(`Bidder ${spec.code} failed to interpret the server's response. Continuing without bids`, null, err);
         requestDone();
         return;
       }
 
+      let bids;
+      if (response && 'bids' in response) {
```

Comment:
I would turn this around to read `if (isArray(response.fledgeAuctionConfigs))`

Because the adapter can technically return a single bid (line 367); while this should work, I think it's less likely that they would accidentally return one that happens to have a `fledgeAuctionConfigs` property that is an array (compared to one that has `bids` of any type).